docs/tutorials/transformer_finetuning.ipynb

```diff
@@ -29,17 +29,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Wanna finetune one of the pretrained transformers for molecular featurization in Molfeat for your property prediction task ? not sure where to start ? Do you need something extra-customized ? Do you feel like getting your hands dirty by hacking Molfeat ? Are you a smooth criminal ? \n",
-    "\n",
-    "**This tutorial is for you !**\n",
-    "\n",
-    "We have previously shown how [Molfeat integrates with PyTorch in general](./integration.html) and even with [Pytorch Geometric](./pyg_integration.html). Now we will demonstrate how to use Molfeat to finetune a pretrained transformer for molecular property prediction.\n",
+    "We have previously shown how [Molfeat integrates with PyTorch in general](./integration.html) and even with [Pytorch Geometric](./pyg_integration.html). Now we will demonstrate how to use Molfeat to finetune a pretrained transformer. This tutorial will walk you through an example of finetuning the ChemBERTa pretrained model for molecular property prediction. These same principles can be applied to any pretrained transformers available in molfeat.\n",
     "\n",
     "To run this tutorial, you will need to install `transformers` and `tokenizers`.\n",
     "\n",
     "`mamba install -c conda-forge transformers \"tokenizers <0.13.2\"`\n",
     "\n",
-    "⚠️ Warning: advanced users only !"
+    "!!! warning\n",
+    "    This tutorial is for advanced users that are comfortable with the APIs of molfeat and Hugging Face transformers."
```

Comment:
This doesn't seem to render for me? I'm not sure it works in a Notebook?

<details><summary>Screenshot</summary>
<p>

![image](https://user-images.githubusercontent.com/28000055/229375468-44820174-362f-48ad-b7da-97b150ceabc9.png)


</p>
</details>